### ISSUE TITLE
Add Somalia holidays

### ISSUE THREAD

**Issue by arkid15r** [vacanza/holidays#1270]
Links:
  - https://en.wikipedia.org/wiki/Public_holidays_in_Somalia
  - https://www.timeanddate.com/holidays/somalia/

**Comment by gtkacz** [comment]
I'd be interested in implementing this. Is it still available?

**Comment by arkid15r** [comment]
> I'd be interested in implementing this. Is it still available?

No, this is part of @Prateekshit73's GSoC proposal. We expect it to be done soon.

**Comment by prateekshit-v** [comment]
## Clarification Needed Regarding Official National Holidays of Somalia

I would like to request clarification regarding the **official national holidays of Somalia**.

I noticed that the list of holidays on the [Somalia Government website](https://www.somalia.gov.so) is not the same as those listed on the [Somalia Embassy in the U.S. website](https://usa.mfa.gov.so/about-somalia/).

Additionally, I came across another source ([archived PDF via Somaliland Law](https://web.archive.org/web/20120729035831/http://www.somalilandlaw.com/Official_Public_Holidays.pdf)) which provides a different holiday list.

Because of these inconsistencies, I’m unsure which calendar should be considered **authoritative and up-to-date**.

Could you please clarify:
1. Which source reflects the **official public holidays of Somalia today**?  
2. Is there a **government gazette/official publication**?

Your guidance would be greatly appreciated 🙏. I just want to ensure I’m following the correct and current information.

Thank you very much!

**Comment by KJhellico** [comment]
Somalia Government website lists "Important events" that are not exactly holidays.

[Somaliland](https://en.wikipedia.org/wiki/Somaliland) <> Somalia. :)
